pkg/build/path_rewriter.go

```diff
@@ -49,5 +46,23 @@ func (rw *RelativePathRewriter) MakeAbs(parent, obj *url.URL) *url.URL {
 	}
 }
 
+// Abs makes an absolute url for URL that has an empty or file-based
+// scheme.
+func (rw *RelativePathRewriter) Abs(obj *url.URL) (*url.URL, error) {
```

Comment:
pass by value as a default would be nicer so I can tell there's no mutation going on.

Reply:
you mean url.URL?

Technically go is always pass by value since references are themselves copied. https://dave.cheney.net/2017/04/29/there-is-no-pass-by-reference-in-go

Reply:
obj is a pointer.   It looks like you're not mutating obj. 
It would be clearer to not pass a pointer, so I know you're not mutating the value of obj. 

Not a big deal.

Reply:
Ack